scripts/bootstrap.js

```diff
@@ -1,5 +1,11 @@
 // @flow
 
+
+// --------------------------------------------- //
```

Comment:
maybe remove it?

Reply:
We'll retain this. Helps in Windows dev environment locally.